files.go

```diff
@@ -176,19 +179,57 @@ func makeStubName(numPart int, namePart string) string {
 	return fmt.Sprintf("%s_%s", zeroPad(numPart, leadingDigits), namePart)
 }
 
+//little utility to read the contents of a list of file names into
+//an array of strings which contains the contents.
+func readFileArray(fileNames []string) ([]string, error) {
+	files := []string{}
+
+	//sort the input array.  This is so fileName_a, fileName _b are sorted in the correct order
+	sort.Strings(fileNames)
+
+	//fwd, err := ioutil.ReadFile(path.Join(dir, name, forwardFile))
+	for _, fileName := range fileNames {
+		bytes, err := ioutil.ReadFile(fileName)
+		if err != nil {
+			return files, err
+		}
+		files = append(files, string(bytes))
+	}
+	return files, nil
+}
+
+// reads the directory containing the folder specified by name.
+// reads all the contents of the file into a Migration.
+// searches directory for all file names containing either "forward"
 func readMigration(dir, name string) (Migration, error) {
 	m := Migration{Name: name}
-	fwd, err := ioutil.ReadFile(path.Join(dir, name, forwardFile))
+	//grab all files that contain word "forward"/"backward"
+	fwdSearch := path.Join(dir, name, "/*forward*.sql")
```

Comment:
You don't need the leading slashes in `/*forward*.sql` and `/*backward*.sql`.  path.Join will insert slashes where necessary.